ui/src/app/store/general/slice.ts

```diff
@@ -8,7 +8,7 @@ const generateInitialState = <K extends keyof GeneralState>(
   initialData: GeneralState[K]["data"]
 ) => ({
   data: initialData,
-  errors: {},
+  errors: null,
```

Comment:
I changed this so that we don't get false positives that the errors exist (because an empty object is truthy - we've made this update for all other models). It doesn't appear that the general errors are being used anywhere (until this branch).